src/overlays/actors/ovl_En_Wiz_Fire/z_en_wiz_fire.c

```diff
@@ -490,7 +490,7 @@ void EnWiz_KillMagicProjectile(EnWizFire* this, PlayState* play) {
 
 void EnWizFire_Update(Actor* thisx, PlayState* play2) {
     // These are AmbientColor, DiffuseColor1, DiffuseColor2, and fogColor
-    static Color_RGB8 lightSettingsColors[] = {
+    static Color_RGB8 sLightSettingsColors[] = {
```

Comment:
How much do we care about updating `variables.txt` for actors that are already decompiled? Because this is in `variables.txt`, but this PR doesn't update the name there. If we don't really care, feel free to ignore this, but if we *do* care, it might make sense to do another pass over this PR to make sure everything's updated in there correctly; I only noticed this one because I named it originally.

Reply:
That'd be a good question for @AngheloAlf since they know more about the build system. If I understand correctly, `variables.txt` might only be relevant when we start other versions? But even then I'm not sure

Reply:
Yeah, at the end of the day, I know that `variables.txt` really doesn't get a lot of attention. The vast majority of variables don't have proper types, and many either don't have the proper names or don't have names at all. So it's not like this PR changes that status quo very much.

Reply:
Updating `variables.txt` will be useful when we start new versions yeah, but not updating it for in-function static and similar variables is fine, they will be easy to catch.